binr/rabin2/rabin2.c

```diff
@@ -577,6 +577,7 @@ int main(int argc, char **argv) {
 		path = r_sys_getenv (R_LIB_ENV);
 		if (path && *path) {
 			r_lib_opendir (l, path);
+			free(path);
```

Comment:
I think executing it always after the `if` is better. Otherwise, if path is not NULL, but it has length == 0, you won't free it.